### ISSUE TITLE
Add links to main TWC site homepage

### ISSUE THREAD

**Issue by jessesquires** [techworkersco/techworkersco.github.io#19]
Depends on #18, once we figure that out. (And once we publish our first new issue.)

Update the links here:
https://github.com/techworkersco/twc-site/blob/develop/_i18n/en.yml#L44-L48

(Also update other languages if possible.)

**Comment by jessesquires** [comment]
#18 is done, so we can do this now

**Comment by jessesquires** [comment]
Alright, this is done. https://github.com/techworkersco/twc-site/pull/217

The PR hasn't landed, but going to close this now since the work is done.
